provider/testdata/recorded/TestProviderUpgrade/compute-firewall/6.67.0/stack.json

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
The new framework expects stack.json instead of state.json ...

Reply:
Bah, my bad. We could rename that back for easier migration if it was a pain?

Reply:
We don't have a big enough deployment base of the old one to bother, I think I can just rename.